pyhoma/models.py

```diff
@@ -285,6 +283,15 @@ def __init__(
         except ValueError:
             self.name = name
 
+        try:
+            self.failure_type_code = (
+                FailureType(failure_type_code) if failure_type_code else None
+            )
+        except ValueError:
+            self.failure_type_code = failure_type_code
+            print(failure_type_code)
+            print(failure_type)
```

Comment:
Instead of a print, we should use a logger.

Reply:
Oops, this was my debug code...